research/picai/nnunet_scripts/predict_and_eval.py

```diff
@@ -0,0 +1,252 @@
+import argparse
+import os
+import time
+import warnings
+from logging import INFO
+from os.path import exists, join
+from typing import Optional
+
+import numpy as np
+from flwr.common.logger import log
+
+from research.picai.nnunet_scripts.eval import get_detection_maps, get_picai_metrics, load_images_from_folder
+from research.picai.nnunet_scripts.predict import predict
+
+
+def pred_and_eval(
+    config_path: str,
+    inputs_folder: str,
+    labels_folder: str,
+    output_folder: Optional[str] = None,
+    save_probability_maps: bool = False,
+    save_detection_maps: bool = False,
+    save_annotations: bool = False,
+    verbose: bool = True,
+) -> None:
+    """
+    Runs inference on raw input data given a number of compatible nnunet
+    models. Then extracts detection maps from those predictions and evaluates
+    the model using the standardPICAI evaluation metrics.
```

Comment:
nit: standardPICAI to standard PICAI

Reply:
If not done elsewhere, maybe worth listing the standard computed metrics in brackets